plugin/trino-cassandra/src/test/java/io/trino/plugin/cassandra/TestCassandraClientConfig.java

```diff
@@ -99,10 +88,6 @@ public void testExplicitPropertyMappings()
                 .put("cassandra.speculative-execution.delay", "101s")
                 .put("cassandra.protocol-version", "V3")
                 .put("cassandra.tls.enabled", "true")
-                .put("cassandra.tls.keystore-path", keystoreFile.toString())
-                .put("cassandra.tls.keystore-password", "keystore-password")
-                .put("cassandra.tls.truststore-path", truststoreFile.toString())
-                .put("cassandra.tls.truststore-password", "truststore-password")
```

Comment:
can you add a test which verifies that if Cassandra connector is started with the old names it still works?

Reply:
Yes I have tested with debugger in the `TestCassandraConnector` and it works for the legacy configs too. But adding a test case will be tricky here, as the connectors bootstrap with valid certificate only.  I don't see any certificates or test cases enabling TLS here.

Reply:
https://github.com/trinodb/trino/pull/20430